doc/releases/changelog-dev.md

```diff
@@ -778,6 +778,9 @@
 
 <h3>Internal changes ⚙️</h3>
 
+* Update docstring in `perturbation_error` to reflect renamed positional arg
+  [#(8174)](https://github.com/PennyLaneAI/pennylane/pull/8174)
+
```

Comment:
Please add it under:

`<h3>Labs: a place for unified and rapid prototyping of research software 🧪</h3>`

Reply:
56c414f

Reply:
[2d6cebb](https://github.com/PennyLaneAI/pennylane/pull/8174/commits/2d6cebb700c0103c05c0a3327a719be6f7cb145d)